src/goInstallTools.ts

```diff
@@ -24,7 +24,8 @@ let tools: { [key: string]: string } = {
 	'go-symbols': 'github.com/newhook/go-symbols',
 	'guru': 'golang.org/x/tools/cmd/guru',
 	'gorename': 'golang.org/x/tools/cmd/gorename',
-	'goimports': 'golang.org/x/tools/cmd/goimports'
+	'goimports': 'golang.org/x/tools/cmd/goimports',
+	'gotests': 'github.com/cweill/gotests'
```

Comment:
This didn't work for me. The source code gets downloaded but the tool doesn't get dropped in the $GOPATH/bin. After looking at the source code structure, looks like there is another "gotests" folder.
This needs to be updated to 'github.com/cweill/gotests/gotests' 

Make the change in the README as well